spec/models/student_registration_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe StudentRegistration do
+  context 'when registering' do
+
+    it 'attempts to save the user'
+    it 'if the save is successful it creates empty project'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.